pages/api/adjudications/edit.js

```diff
@@ -15,13 +15,13 @@ export default async (req, res) => {
       cumulativeMark,
       audioUrl,
       notes,
-      specialAward,
       performanceID,
     } = req.body;
 
     // If all params exist
     if (id && artisticMark && technicalMark && cumulativeMark && performanceID) {
       // Update adjudication
+      // TODO: set user_id from session???
```

Comment:
adjudication now should store data about the user who created the adjudication. Should I be getting the id from session or will it be passed from the frontend

Reply:
Assume passed for now.  The frontend will/should know the user, user id, user name and user email.

Reply:
@feng-eric  actually, scrap that.  Let's use the session in the backend to find out all the user details.  I think that's probably better.